### ISSUE TITLE
Ensure all text has a contrast ratio of 4.5:1

### ISSUE THREAD

**Issue by LucyGwilliamAdmin** [open-sdg/open-sdg#848]
**Page 123 in DAC report**

**Issue from DAC report:**
Some areas of default contrast site don't pass colour contrast rules and some areas of high contrast site don't pass colour contrast rules. This means users have to switch back and forth. See images for examples.

**Solution from DAC report:**
Please ensure that the visual presentation of text has a colour contrast ratio of at least 4.5:1. This can be done by selecting a darker background colour or adjusting the colour of the text.

Note:
For sites to pass AA, they must comply with WCAG 2.1 checkpoint 1.4.3 for colour contrast, which specifies the following minimum contrast ratios:

- if text is not bold and its size is less than 18pt: 4.5:1 for AA level;
- if text is not bold and its size is at least 18pt: 3:1 for AA level;
- if text is bold and its size is less than 14pt: 4.5:1 for AA level;
- if text is bold and its size is at least 14pt: 3:1 for AA level

**Example from DAC report:**
No example provided

![image](https://user-images.githubusercontent.com/43404358/91591727-422da100-e955-11ea-809c-287e79ac8ce6.png)
![image](https://user-images.githubusercontent.com/43404358/91591732-46f25500-e955-11ea-94bd-a4e7c37dda6b.png)

**Comment by brockfanning** [comment]
Looking at the high-contrast screenshots I believe those should be fixed by #865. I am not sure what is being pointed out in the normal-contrast screenshots.

**Comment by brockfanning** [comment]
@LucyGwilliamAdmin Now that #865 is merged, I wondering if some/all of this may be fixed. Can you look at a recent feature and see if maybe this could be closed?

**Comment by LucyGwilliamAdmin** [comment]
@brockfanning I know we discussed in a recent tech call and I agreed we could close this one but I've just noticed the table title (caption) doesn't have a contrast ratio of 4.5:1 when in default contrast mode.

Background is `#FFFFFF` and text is `#777` giving it a contrast ratio of 4.47:1

![image](https://user-images.githubusercontent.com/43404358/93612407-5f3f1800-f9c7-11ea-80b8-ff5ec4d34fad.png)

Thinking we could make it consistent with rest of main colour `#0b0c0c` so that it would look like this:
![image](https://user-images.githubusercontent.com/43404358/93612869-fdcb7900-f9c7-11ea-8954-74cc0165f788.png)

**Comment by LucyGwilliamAdmin** [comment]
Have just merged PR which makes table caption same colour as rest of text so closing again
